imod/mf6/riv.py

```diff
@@ -130,6 +130,7 @@ class River(BoundaryCondition):
         "concentration": (RegridderType.OVERLAP, "mean"),
     }
 
+    @pkg_init(exclude_in_dataset=["validate"])
```

Comment:
this makes me wonder how the wrapping will work if we create derived types of the river class. Will we automatically get the same wrapper when we call super(*).__init__ ? can we remove the wrapper if we don't want it? if the new class also uses the @pkg_init wrapper, will it be done twice?